easybuild/easyconfigs/x/xalt/xalt-2.7.27-gpu.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'EB_XALT'
+
+name = 'xalt'
+version = '2.7.27'
+# This does not work since XALT automatically adds xalt/<version> to
+# the install prefix and does not recognize xalt/<version>-gpu.
+#versionsuffix = '-gpu'
+
+homepage = 'https://github.com/xalt/xalt'
+description = """XALT 2 is a tool to allow a site to track user executables
+  and library usage on a cluster. When installed it can tell a site what are
+  the top executables by Node-Hours or by the number of users or the number
+  of times it is run. XALT 2 also tracks library usage as well. XALT 2 can
+  also track package use by R, MATLAB or Python. It tracks both MPI and
+  non-MPI programs."""
+
+toolchain = SYSTEM
+
+# The XALT site filter config file is site specific and must be specified
+# to successfully build XALT.  Use "--try-amend=config_py=<file>" when
+# building this EasyConfig to specify the site filter config file.
+config_py = None
+
+# Extra options
+#syshost = 'hardcode:cluster'
+#transmission = 'syslog'
+
+configopts = '--with-MySQL=no --with-trackGPU=yes'
+
+source_urls = ['https://github.com/xalt/xalt/archive/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['d0dcefe5d8b342ca232bd1ce8ae675fed714591e49fa96eef9a13b392786b5d8']
+
+builddependencies = [
+    ('CUDA', '10.1.243'),
+    ('cURL', '7.60.0-GCCcore-7.3.0'),
+    ('flex', '2.6.4-GCCcore-7.3.0'),
+    ('LibUUID', '1.0.3-GCCcore-7.3.0'),
+]
+
+osdependencies = [
+    ('bc'),
+    ('coreutils'),
+    ('libelf-dev', 'elfutils-libelf-devel'),
+    ('libssl-dev', 'openssl-devel'),
+    ('libxml2'), # required for CUDA-10.1.243
+]
+
+sanity_check_paths = {
```

Comment:
@samcmill This can/should be moved to the `XALT` easyblock too, by customizing `sanity_check_step`